src/panels/config/areas/ha-config-area-page.ts

```diff
@@ -0,0 +1,399 @@
+            ${isComponentLoaded(this.hass, "automation")
+              ? html`
+                  <ha-card
+                    .header=${this.hass.localize(
+                      "ui.panel.config.devices.automation.automations"
+                    )}
+                    >${this._related?.automation?.length
+                      ? this._related.automation.map((automation) => {
+                          const state = this.hass.states[automation];
+                          return state
+                            ? html`
+                                <div>
+                                  <paper-item
+                                    .automation=${state}
+                                    @click=${this._openAutomation}
+                                    .disabled=${!state.attributes.id}
+                                  >
+                                    <paper-item-body>
+                                      ${state.attributes.friendly_name ||
+                                        automation}
+                                    </paper-item-body>
+                                    <ha-icon-next></ha-icon-next>
+                                  </paper-item>
+                                  ${!state.attributes.id
+                                    ? html`
+                                        <paper-tooltip
+                                          >${this.hass.localize(
+                                            "ui.panel.config.devices.cant_edit"
+                                          )}
+                                        </paper-tooltip>
+                                      `
+                                    : ""}
+                                </div>
+                              `
+                            : "";
+                        })
+                      : html`
+                          <paper-item class="no-link"
+                            >${this.hass.localize(
+                              "ui.panel.config.devices.automation.no_automations"
+                            )}</paper-item
+                          >
+                        `}
+                  </ha-card>
+                `
+              : ""}
+          </div>
+          <div class="column">
+            ${isComponentLoaded(this.hass, "scene")
+              ? html`
+                  <ha-card
+                    .header=${this.hass.localize(
+                      "ui.panel.config.devices.scene.scenes"
+                    )}
+                    >${this._related?.scene?.length
+                      ? this._related.scene.map((scene) => {
+                          const state = this.hass.states[scene];
+                          return state
+                            ? html`
+                                <div>
+                                  <paper-item
+                                    .scene=${state}
+                                    @click=${this._openScene}
```

Comment:
Instead of using `@click`, wrap `<paper-item>` in an `<a>` tag. The benefit of this is that you can now also open these links in a new tab. We should do that for all cases everywhere where we are just opening a new page on click.